authentication/auth.go

```diff
@@ -251,7 +261,28 @@ func launcherSchemaFromURL(url string) (launcherSchema surveys.LauncherSchema) {
 		URL:      url + cacheBust,
 	}
 
-	return launcherSchema
+	return launcherSchema, ""
+}
+
+func validateSchema(payload []byte) (error string) {
+	if settings.Get("SCHEMA_VALIDATOR_URL") != "" {
```

Comment:
invert `if` to reduce nesting?

Reply:
If I do that I will have to duplicate the `return ""`